src/components/StoreList/actions.js

```diff
@@ -4,6 +4,8 @@ import {
 	SET_SHOW_LIST
 } from './actionTypes';
 
+import getStore from '../../utils/honestyStore.js';
```

Comment:
Since this is a refactor PR could we rename this? It's easy to get confused if we're talking about the redux store, honesty store etc.

What is it really doing? Fetching items? `fetchItems` maybe?